internal/http/services/owncloud/ocs/handlers/cloud/capabilities/uploads.go

```diff
@@ -49,32 +49,41 @@ func (h *Handler) getCapabilitiesForUserAgent(_ context.Context, userAgent strin
 }
 
 func setCapabilitiesForChunkProtocol(cp chunkProtocol, c *data.Capabilities) {
-	switch cp {
-	case chunkV1:
-		// 2.7+ will use Chunking V1 if "capabilities > files > bigfilechunking" is "true" AND "capabilities > dav > chunking" is not there
-		c.Files.BigFileChunking = true
-		c.Dav = nil
-		c.Files.TusSupport = nil
+	// 2.7+ will use Chunking V1 if "capabilities > files > bigfilechunking" is "true" AND "capabilities > dav > chunking" is not there
+	c.Files.BigFileChunking = true
+	c.Dav = nil
+	c.Files.TusSupport = nil
+	/*
```

Comment:
Do we need to keep the comment?

Reply:
Good question, I'd say yes for now if we have to resurrect support for TUS.